.travis.yml

```diff
@@ -91,6 +91,7 @@ script:
       make clean_travis &&
       make package &&
       make -C tools/plist_to_html test &&
+      make -C tools/tu_collector test &&
```

Comment:
We run the tu_collector test cases multiple times:
https://github.com/Ericsson/codechecker/blob/c8fd5d24d2f99fb7d124046f5243099cd1520d5f/.travis.yml#L96

Reply:
I kept the linux only test for now, the tu_collector depends on gcc/g++ later (#2260) when it will work with clang we can enable the tests for both environments.